src/pages/index.js

```diff
@@ -0,0 +1,285 @@
+    });
+    profileName.textContent = userInfo.name;
+    profileAvatar.alt = userInfo.name;
+    profileDescription.textContent = userInfo.about;
+  })
+  .catch(console.error);
+
+function handleDeleteCard(cardElement, cardId) {
+  selectedCard = cardElement;
+  selectedCardId = cardId;
+  openModal(deleteModal);
+}
+function getCardElement(data) {
+  const cardElement = cardTemplate.content
+    .querySelector(`.card`)
+    .cloneNode(true);
+  const cardDeleteBtn = cardElement.querySelector(`.card__delete-btn`);
+  const cardNameElement = cardElement.querySelector(`.card__title`);
+  const cardImageElement = cardElement.querySelector(`.card__image`);
+  const cardLikeBtn = cardElement.querySelector(`.card__like-button`);
+
+  cardNameElement.textContent = data.name;
+  cardImageElement.src = data.link;
+  cardImageElement.alt = data.name;
+
+  if (data.isLiked) {
+    cardLikeBtn.classList.add("card__like-button_liked");
+  }
+
+  cardLikeBtn.addEventListener("click", (evt) => {
+    handleCardLike(evt, data._id);
+  });
+
+  cardDeleteBtn.addEventListener("click", () =>
+    handleDeleteCard(cardElement, data._id)
+  );
+
+  cardImageElement.addEventListener("click", () => {
+    openModal(previewModal);
+    previewModalImage.src = data.link;
+    previewModalImage.alt = data.name;
+    previewModalPicDescription.textContent = data.name;
+  });
+  return cardElement;
+}
+function handleCardLike(evt, id) {
+  const isLiked = evt.target.classList.contains("card__like-button_liked");
+  api
+    .changeLikeStatus(id, isLiked)
+    .then(() => {
+      evt.target.classList.toggle("card__like-button_liked");
+    })
+    .catch(console.error);
+}
+function openModal(modal) {
+  modal.classList.add("modal_opened");
+  document.addEventListener("keydown", closeModalEscape);
+}
+function closeModal(modal) {
+  modal.classList.remove("modal_opened");
+  document.removeEventListener("keydown", closeModalEscape);
+}
+previewModalCloseBtn.addEventListener("click", () => {
+  closeModal(previewModal);
+});
+function handleEditFormSubmit(evt) {
+  evt.preventDefault();
+
+  const submitBtn = evt.submitter;
+  setButtonText(submitBtn, true);
+  api
+    .editUserInfo({
+      name: editModalNameInput.value,
+      about: editModalDescriptionInput.value,
+    })
+    .then((data) => {
+      profileName.textContent = data.name;
+      profileDescription.textContent = data.about;
+    })
+    .catch(console.error)
+    .finally(() => {
+      setButtonText(submitBtn, !true);
+    });
+}
+function handleCardFormSubmit(evt) {
+  evt.preventDefault();
+  const inputValues = {
+    name: cardModalNameInput.value,
+    link: cardModalLinkInput.value,
+  };
+  const submitBtn = evt.submitter;
+  setButtonText(submitBtn, true);
+  api
+    .postInitialCards(inputValues)
+    .then((data) => {
+      previewModalPicDescription.textContent = data.name;
+      previewModalImage.src = data.link;
+      previewModalImage.alt = data.name;
```

Comment:
```js

      previewModalPicDescription.textContent = data.name;
      previewModalImage.src = data.link;
      previewModalImage.alt = data.name;
```

You don't need it when you add a new card. It's done only on the image modal opening (after clicking on the image)

Reply:
Thank you I made the necessary corrections by removing the strings from 170-172